nntrainer/tensor/float_tensor.cpp

```diff
@@ -493,8 +493,8 @@ void FloatTensor::sum_by_batch(Tensor &output) const {
 
   Tensor ones(1, 1, 1, feat_len, this->getFormat());
   ones.setValue(1.0);
-  sgemv(CblasRowMajor, CblasNoTrans, batch, feat_len, 1, data, feat_len,
-        ones.getData<float>(), 1, 0.0, out_data, 1);
+  sgemv((unsigned int)dim.getStorageOrder(), false, batch, feat_len, 1, data,
```

Comment:
This is just a suggestion! how about having a fixed value for storage orders like transpose?
although there's no difference in the result, I think it would be easier for us to understand the code and debug.

Reply:
I don't really get it... Could you elaborate a little bit more for me?
I think current implementation is quite similar to transpose cases.
With my understanding of your suggestion, do you mean we should have functions like:
```
sgemv_rowMaj(...)
...
sgemv_colMah(...)
....
```
?

Reply:
what I meant by having a fixed value is as follows.
```suggestion
  sgemv(TStorageOrder::ROW_MAJOR, false, batch, feat_len, 1, data,
```
same as we pass the transpose with true/false!